### ISSUE TITLE
Add AutoCreate and Database Groups arguments to wr.get_redshift_temp_engine()

### ISSUE THREAD

**Issue by acere** [awslabs/aws-data-wrangler#288]
**Problem**

`get_redshift_temp_engine()` does not allow to pass  `AutoCreate` and `DbGroups` to the underlying `client_redshift.get_cluster_credentials()`, making it impossible to authenticate temporary users, as described in [here](https://docs.aws.amazon.com/redshift/latest/mgmt/generating-iam-credentials-cli-api.html).

**Suggested Solution**

Include the optional arguments `DBGroups` and `AutoCreate` to `get_redshift_temp_engine()`.

**Comment by igorborgest** [comment]
Thanks @acere, cool suggestion !

I've just add it in our development branch, and will be released soon on version `1.6.0`.

Could you test it, and give us some feedback?

To install the development branch:

```
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

Example:

```py3
engine = wr.db.get_redshift_temp_engine(  # auto_create is True by default
    cluster_identifier="my-cluster",
    user="john_doe",
    db_groups=["my-group"]
)
with engine.connect() as con:
    cursor = con.execute("SELECT 1")
    print(cursor.fetchall())
```

**Comment by igorborgest** [comment]
Released on version [1.6.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.6.0)!

**Comment by acere** [comment]
> Released on version [1.6.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.6.0)!

Thanks for the quick change!

**Comment by igorborgest** [comment]
Hi @acere !

It is totally off-topic, but we are stating a [_"Who uses AWS Data Wrangler?"_](https://github.com/awslabs/aws-data-wrangler/tree/dev#who-uses-aws-data-wrangler) section. So feel free to add yourself if you want 😄 .
